engine/commands/model_import_cmd.cc

```diff
@@ -16,15 +16,15 @@ ModelImportCmd::ModelImportCmd(std::string model_handle, std::string model_path)
 void ModelImportCmd::Exec() {
   config::GGUFHandler gguf_handler;
   config::YamlHandler yaml_handler;
-  modellist_utils::ModelListUtils modellist_utils_obj;
+  cortex::db::Models modellist_utils_obj;
 
   std::string model_yaml_path = (file_manager_utils::GetModelsContainerPath() /
                                  std::filesystem::path("imported") /
                                  std::filesystem::path(model_handle_ + ".yml"))
                                     .string();
-  modellist_utils::ModelEntry model_entry{
+  cortex::db::ModelEntry model_entry{
       model_handle_,   "local",       "imported",
-      model_yaml_path, model_handle_, modellist_utils::ModelStatus::READY};
+      model_yaml_path, model_handle_, cortex::db::ModelStatus::READY};
```

Comment:
maybe we should remove `cortex::db::ModelStatus::READY`.

Reply:
Removed it